app/controllers/coaches_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CoachesController < ApplicationController
+  def index
+  end
+
+  def show
+  end
+
+  private
+  helper_method :chapters
+  def chapters
+    @chapters ||= Chapter.by_name
+  end
+
+  helper_method :chapter
+  def chapter
+    @chapter ||= chapters.find_by!(slug: params[:chapter_slug])
+  end
+
+  helper_method :coaches
+  def coaches
+    @coaches ||= chapter.coaches.sort_by do |coach|
+      coach.attended_sessions.count
+    end.reverse
```

Comment:
I'm not crazy happy about this approach to loading coaches sorted by attended sessions (It's N+2), so there is room to move this over to a single SQL query if we need.